hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/TestHoodieSparkSqlWriter.scala

```diff
@@ -302,9 +306,10 @@ class HoodieSparkSqlWriterSuite {
       try {
         // write to Hudi
         HoodieSparkSqlWriter.write(sqlContext, SaveMode.Append, fooTableModifier, df)
-        Assertions.fail("Should have thrown exception")
+        fail("Should have thrown exception")
       } catch {
-        case e: HoodieException => assertTrue(e.getMessage.contains("hoodie.populate.meta.fields already disabled for the table. Can't be re-enabled back"))
+        case e: HoodieException => assertTrue(e.getMessage.startsWith("Config conflict"))
```

Comment:
@nsivabalan please note that conflict config like `populate MetaFields` will be captured in `org.apache.hudi.HoodieWriterUtils#validateTableConfig`